lib/cursor.js

```diff
@@ -1,7 +1,36 @@
-import { go, put } from '../vendor/js-csp';
-import { partial } from 'underscore';
+import { isObject, isFunction, partial, defaults, map, filter, find, isArray } from 'underscore';
 import ObjectPath from 'object-path';
+import putOnChan from './put_on_chan';
 
+const hasSameValue = function (valueA, cursorB) {
+  return isEqual(valueA, cursorB.value);
```

Comment:
this is in the cursor file and the method is used on a cursor. meant for this usage:
```javascript
shouldComponentUpdate: function (nextProps, nextState) {
  return this.props.cursor.hasSameValue(nextProps.cursor);
}
```